recipes/jxrlib/all/patches/0001-missing-declarations.patch

```diff
@@ -0,0 +1,26 @@
+--- a/image/decode/segdec.c
++++ b/image/decode/segdec.c
+@@ -52,6 +52,13 @@ static Int DecodeSignificantAbsLevel (struct CAdaptiveHuffman *pAHexpt, BitIOInf
+ //================================================================
+ // Memory access functions
+ //================================================================
++#ifndef _BIG__ENDIAN_
++#if (defined(WIN32) && !defined(UNDER_CE)) || (defined(UNDER_CE) && defined(_ARM_))
++// WinCE ARM and Desktop x86
++#else
++U32 _byteswap_ulong(U32 bits);
++#endif
++#endif
```

Comment:
for the review:

- this compilation unit calls `_byteswap_ulong` if `_BIG_ENDIAN` not defined.
- I've reproduced logic from `strcodec.c` (where this function might be defined) to know when to declarare this function: https://github.com/4creators/jxrlib/blob/f7521879862b9085318e814c6157490dd9dbbdb4/image/sys/strcodec.c#L667-L684